### ISSUE TITLE
Pre-work Checklist: Developer: Vincent Dang

### ISSUE THREAD

**Issue by vincentdang** [hackforla/website#3830]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by ExperimentsInHonesty** [comment]
@vincentdang How are you doing on setting up your development environment?  It looks like you have not yet assigned yourself to a good first issue.  If you need help please reach out in the hfla-site slack channel or come to a team meeting.

**Comment by jdingeman** [comment]
@vincentdang - hey Vincent, left you a message on Slack. If we don't hear from you by 03-30-2023, then we will assume you have left the team. Please let us know how we can support you.

**Comment by roslynwythe** [comment]
@vincentdang  Since we have not heard from you. I am going to go ahead and close this issue. If you want to return to the website team in the future, feel free to reopen this issue and put it in the in progress column.

**Comment by vincentdang** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings? 
- It is hard to recall but probably up to two weeks or more due to my inconsistency and being busy.

**Comment by roslynwythe** [comment]
Hi @vincentdang it looks like you've made good progress completing issues and you are currently assigned to a small issue.  Please also leave a comment in this issue on a weekly basis, as described above https://github.com/hackforla/website/issues/3830#issuecomment-1491428513

If an issue is inactive for more than 2 weeks, it is flagged for review by a dev lead.  Please follow instructions in the comment above, for leaving a progress report and removing the "2 weeks inactive" label.  

Thank you!  We really appreciate your contributions.

**Comment by vincentdang** [comment]
@roslynwythe Apologies for the inconsistencies, yea I'm good at doing issues but I'm bad at keeping up with my pre-work.  

Progress: Currently awaiting the merge team to respond to an issue https://github.com/hackforla/website/issues/4423
Blockers: Busy with work but mainly awaiting the merge team to respond.
Availability: I can work on this at night time during weekdays.
ETA: Unknown.

**Comment by vincentdang** [comment]
I'm done with my pre-work.

**Comment by roslynwythe** [comment]
Hi @vincentdang our process now is to keep the prework issues open indefinately, as a means of tracking developer activity and communicating with developers.   It appears that you have not been active in Hack for LA since April 2023.  During periods when you are not working on any issues, we request that you post a comment in this issue to inform us of your progress, any problems you might be having, and to make sure that you are still interested in continuing with Hack for LA.  We would like to see updates, in this issue or in other issues, at least every two weeks.  

If we do not hear from you by 7/10/2023, we will assume that you are not interested in continuing with Hack for LA and we will close this issue, but you can reopen it at any time.  If you do reopen the issue, please move it into the "In Progress" column.  Thank you.

**Comment by vincentdang** [comment]
Update: Finished one issue. Working on a second issue.  Was busy with work and other stuff.  Will continue to work on issue when possible.

**Comment by vincentdang** [comment]
Update: Made progress on second issue.  First issue is still pending.

**Comment by vincentdang** [comment]
Update: Made progress on second issue. First issue is still pending.

**Comment by vincentdang** [comment]
Update: All issues fixed and closed. 
Progress: Will begin seeking new issue related to backend development.

**Comment by vincentdang** [comment]
Decided to move on from the website and begin attempting backend work with tdm-calculator.

**Comment by HackforLABot** [comment]
## Activity Log: vincentdang
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- vincentdang opened: https://github.com/hackforla/website/issues/3830 at 01/16/2023, 08:17 PM PST
- vincentdang assigned: https://github.com/hackforla/website/issues/3830 at 01/16/2023, 08:28 PM PST
- vincentdang commented on: https://github.com/hackforla/website/pull/3836#issuecomment-1386460013 at 01/17/2023, 08:11 PM PST
- vincentdang commented on: https://github.com/hackforla/website/pull/3836#issuecomment-1386488069 at 01/17/2023, 08:56 PM PST
- vincentdang commented on: https://github.com/hackforla/website/pull/3836#issuecomment-1386490126 at 01/17/2023, 09:00 PM PST
- vincentdang submitted review: https://github.com/hackforla/website/pull/3836#pullrequestreview-1264754587 at 01/22/2023, 03:41 AM PST
- vincentdang issue closed by PR 4273: https://github.com/hackforla/website/issues/2833#event-8441846519 at 02/05/2023, 02:18 PM PST
- vincentdang assigned: https://github.com/hackforla/website/issues/2833#issuecomment-1451051775 at 03/23/2023, 01:47 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/2833#issuecomment-1481878672 at 03/23/2023, 01:50 PM PDT
- vincentdang opened: https://github.com/hackforla/website/pull/4266 at 03/23/2023, 05:54 PM PDT
- vincentdang opened: https://github.com/hackforla/website/pull/4273 at 03/23/2023, 08:02 PM PDT
- vincentdang PR was closed: https://github.com/hackforla/website/pull/4266#event-8846112799 at 03/25/2023, 03:17 PM PDT
- vincentdang issue closed as completed: https://github.com/hackforla/website/issues/3830#event-8928409357 at 04/04/2023, 01:12 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/pull/4273#issuecomment-1498447866 at 04/05/2023, 08:40 PM PDT
- vincentdang PR was merged: https://github.com/hackforla/website/pull/4273#event-8941439565 at 04/05/2023, 09:15 PM PDT
- vincentdang issue closed by PR 4273: https://github.com/hackforla/website/issues/2833#event-8941439658 at 04/05/2023, 09:15 PM PDT
- vincentdang assigned: https://github.com/hackforla/website/issues/4250 at 04/06/2023, 11:00 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4250#issuecomment-1499976135 at 04/06/2023, 11:13 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4250#issuecomment-1504293344 at 04/11/2023, 05:00 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/3830#issuecomment-1504802927 at 04/12/2023, 12:33 AM PDT
- vincentdang reopened: https://github.com/hackforla/website/issues/3830#issuecomment-1504802927 at 04/12/2023, 12:35 AM PDT
- vincentdang commented on: https://github.com/hackforla/website/pull/4266#issuecomment-1504808668 at 04/12/2023, 12:38 AM PDT
- vincentdang issue closed as completed: https://github.com/hackforla/website/issues/4250#event-9030468188 at 04/17/2023, 06:36 PM PDT
- vincentdang assigned: https://github.com/hackforla/website/issues/4423 at 04/23/2023, 03:24 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4423#issuecomment-1519183862 at 04/23/2023, 03:34 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4423#issuecomment-1521134624 at 04/24/2023, 09:20 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4423#issuecomment-1524043687 at 04/26/2023, 02:04 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4423#issuecomment-1528384389 at 04/28/2023, 06:19 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4423#issuecomment-1537692926 at 05/07/2023, 08:39 PM PDT
- vincentdang issue closed as completed: https://github.com/hackforla/website/issues/4485#event-9217884689 at 05/10/2023, 10:00 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/3830#issuecomment-1546741669 at 05/13/2023, 01:07 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4423#issuecomment-1554156541 at 05/19/2023, 12:25 AM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4423#issuecomment-1555056670 at 05/19/2023, 11:15 AM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4423#issuecomment-1555060489 at 05/19/2023, 11:19 AM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/3830#issuecomment-1556091912 at 05/20/2023, 10:59 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4423#issuecomment-1565934674 at 05/27/2023, 11:34 PM PDT
- vincentdang assigned: https://github.com/hackforla/website/issues/4793 at 06/22/2023, 05:08 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4793#issuecomment-1603463448 at 06/22/2023, 05:12 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4793#issuecomment-1613949157 at 06/29/2023, 05:24 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4793#issuecomment-1622854484 at 07/05/2023, 07:19 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4793#issuecomment-1627573737 at 07/08/2023, 07:12 PM PDT
- vincentdang issue closed as completed: https://github.com/hackforla/website/issues/4793#event-9768190965 at 07/09/2023, 04:43 PM PDT
- vincentdang assigned: https://github.com/hackforla/website/issues/4485 at 07/11/2023, 05:38 AM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4485#issuecomment-1630752807 at 07/11/2023, 05:40 AM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/3830#issuecomment-1632784490 at 07/12/2023, 08:46 AM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4485#issuecomment-1634387481 at 07/13/2023, 07:52 AM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4485#issuecomment-1635090336 at 07/13/2023, 05:33 PM PDT
- vincentdang opened: https://github.com/hackforla/website/pull/4952 at 07/16/2023, 06:56 AM PDT
- vincentdang closed: https://github.com/hackforla/website/pull/4952#event-9831187029 at 07/16/2023, 06:59 AM PDT
- vincentdang commented on: https://github.com/hackforla/website/pull/4952#issuecomment-1637091971 at 07/16/2023, 07:00 AM PDT
- vincentdang opened: https://github.com/hackforla/website/pull/4953 at 07/16/2023, 07:06 AM PDT
- vincentdang commented on: https://github.com/hackforla/website/pull/4651#issuecomment-1637143248 at 07/16/2023, 10:17 AM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4485#issuecomment-1637152671 at 07/16/2023, 11:00 AM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4485#issuecomment-1637154465 at 07/16/2023, 11:10 AM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/3830#issuecomment-1637263939 at 07/16/2023, 06:24 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4485#issuecomment-1639274719 at 07/17/2023, 08:22 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4485#issuecomment-1656512312 at 07/28/2023, 06:48 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/pull/4953#issuecomment-1656516050 at 07/28/2023, 07:00 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4485#issuecomment-1656517585 at 07/28/2023, 07:06 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4485#issuecomment-1656518240 at 07/28/2023, 07:08 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/3830#issuecomment-1656519669 at 07/28/2023, 07:12 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4485#issuecomment-1659421120 at 07/31/2023, 06:17 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/4485#issuecomment-1663100424 at 08/02/2023, 04:40 PM PDT
- vincentdang issue closed as completed: https://github.com/hackforla/website/issues/3830#event-10029720265 at 08/07/2023, 05:26 PM PDT
- vincentdang opened: https://github.com/hackforla/website/pull/5162 at 08/07/2023, 07:09 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/pull/4953#issuecomment-1668804415 at 08/07/2023, 07:10 PM PDT
- vincentdang PR was merged: https://github.com/hackforla/website/pull/5162#event-10049709263 at 08/09/2023, 09:13 AM PDT
- vincentdang PR was closed: https://github.com/hackforla/website/pull/4953#event-10049755042 at 08/09/2023, 09:18 AM PDT
- vincentdang issue closed as completed: https://github.com/hackforla/website/issues/4485#event-10070113724 at 08/11/2023, 07:02 AM PDT
- vincentdang commented on: https://github.com/hackforla/website/pull/5162#issuecomment-1676236683 at 08/12/2023, 10:35 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/3830#issuecomment-1676237230 at 08/12/2023, 10:38 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/2228#issuecomment-1683220497 at 08/17/2023, 06:55 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/3229#issuecomment-1684546506 at 08/18/2023, 04:27 PM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/2228#issuecomment-1688466952 at 08/22/2023, 08:46 AM PDT
- vincentdang commented on: https://github.com/hackforla/website/issues/3830#issuecomment-1801160796 at 11/07/2023, 10:18 PM PST
